src/main/java/com/scalefocus/blog_api/controller/UserController.java

```diff
@@ -48,6 +53,7 @@ public ResponseEntity<TokenResponse> register(@RequestBody RegisterRequest regis
     )
     @PostMapping("/authenticate")
     public ResponseEntity<TokenResponse> login(@RequestBody AuthenticationRequest authenticationRequest) {
+        logger.info("User '{}' is trying to login to the system", authenticationRequest.getUsername());
```

Comment:
It is better to log Ids instead of usernames. This way if the log is leaked, it will be meaningless.

Reply:
Done